docs/development/extensions-core/datasketches-hll.md

```diff
@@ -89,6 +94,11 @@ druid.extensions.loadList=["druid-datasketches"]
  }
 ```
 
+The `HLLSketchMerge` aggregator can be used to ingest pre-generated sketches from an input dataset. For example, an
+earlier batch processing job can be used to generate the sketches before the data is sent to Druid. To support this
+behaviour, the sketches in the input dataset must be serialised to base64-encoded bytes. Then, in the native ingestion
+`MetricsSpec` the `HLLSketchMerge` must be specified for the input column as shown above.
+
```

Comment:
```suggestion
You can use the `HLLSketchMerge` aggregator to ingest pre-generated sketches from an input dataset. For example, you can set up a batch processing job to generate the sketches before sending the data to Druid. You must serialize the sketches in the input dataset to base-64 encoded bytes. Then, specify `HLLSketchMerge` for the input column in the native ingestion`MetricsSpec`.

```
Stylistic suggestions. Also wonder if it might be helpful to have an example of the `MetricsSpec`.

Reply:
I'm not sure on the usefulness of an example - the code blob to be added to the `MetricsSpec` is already included directly above this paragraph.